src/main/java/bro/Parser.java

```diff
@@ -0,0 +1,71 @@
+package bro;
+
+import bro.command.*;
+import bro.task.Deadline;
+import bro.task.Event;
+import bro.task.Todo;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class Parser {
+
+    public Command parse(String str) throws BroException {
+
+        this.checkEmptyInput(str);
+        String[] in = str.split(" ");
+        String input = in[0];
+            switch (input) {
+                case "list":
```

Comment:
Perhaps you can remove the indentation for case clauses to follow the Java coding standard.

Reply:
I have made the changes!!
Thank you

Reply:
Btw I requested a re-review by mistake!
sorry